### PR TITLE
feat(core): add locale header

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: packages/manager/modules/core/src/index.js [hunk 1]
```diff
@@ -185,6 +187,9 @@ angular
     $httpProvider.interceptors.push('OvhSsoAuthInterceptor');
     $httpProvider.interceptors.push('OvhNgRequestTaggerInterceptor');
   })
+  .config(($httpProvider) => {
```
Comment (antleblanc) [inline packages/manager/modules/core/src/index.js]: Explicit annotation could be added:

```suggestion
  .config(/* @ngInject */($httpProvider) => {
```

**Comment by FredericEspiau** [comment]
Why ?

**Comment by marie-j** [comment]
> Why ?

@FredericEspiau Locale is needed in a 2API but instead of passing a query param each time we will need it, we just need to get the header. I don't know if that makes more sense. 
A benefit from this we can imagine is we want API (or 2API) to sort something depending on user's language, we don't need to get and pass it each time, it is only done at one place and avoids duplication

**Comment by antleblanc** [comment]
@marie-j Why not considering using this information by doing:

```txt
GET /me
{
  "language": "..."
}
```

**Comment by FredericEspiau** [comment]
£is it not possible to use `Content-Language` instead ? It's the header intended for this usage

File: packages/manager/modules/core/src/translate/translate.constants.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const HEADER_LOCALE = 'Content-Language';
```
Comment (FredericEspiau) [inline packages/manager/modules/core/src/translate/translate.constants.js]: Shouldn't we have a custom header instead ? It would give us more control !

Comment (FredericEspiau) [reply packages/manager/modules/core/src/translate/translate.constants.js]: (I'm kidding)

Comment (marie-j) [reply packages/manager/modules/core/src/translate/translate.constants.js]: 😭

**Pull request merged**